applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -116,6 +116,16 @@ def GetDefaultParameters(cls):
             "line_search": false,
             "use_old_stiffness_in_first_iteration": false,
             "compute_reactions": true,
+            "solving_strategy_settings": {
+                "strategy_type" : "newton_raphson",
```

Comment:
```suggestion
                "type" : "newton_raphson",
```

these are the solving_strategy settings, hence should be clear what is meant :)